packages/puppeteer-core/src/node/ScreenRecorder.ts

```diff
@@ -174,30 +220,59 @@ export class ScreenRecorder extends PassThrough {
     );
   }
 
-  #getFormatArgs(format: 'webm' | 'gif') {
+  #getFormatArgs(
+    format: FileFormat,
+    fps: number | 'source_fps',
+    loop: number,
+    delay: number,
+    quality: number,
+    colors: number,
+  ) {
+    const libvpx = [
+      // Sets the codec to use.
+      ['-c:v', 'vp9'],
+      // Sets the quality. Lower the better.
+      ['-crf', `${quality}`],
+      // Sets the quality and how efficient the compression will be.
+      ['-deadline', 'realtime', '-cpu-used', `${os.cpus().length / 2}`],
+    ];
     switch (format) {
       case 'webm':
         return [
-          // Sets the codec to use.
-          ['-c:v', 'vp9'],
+          ...libvpx,
           // Sets the format
           ['-f', 'webm'],
-          // Sets the quality. Lower the better.
-          ['-crf', `${CRF_VALUE}`],
-          // Sets the quality and how efficient the compression will be.
-          ['-deadline', 'realtime', '-cpu-used', '8'],
         ].flat();
       case 'gif':
+        fps = DEFAULT_FPS === fps ? 20 : 'source_fps';
+        if (loop === Infinity) {
+          loop = 0;
+        }
+        if (delay !== -1) {
+          delay /= 10;
+        } // ms to cs
```

Comment:
Can you move this comment above the the delay /=10;

Reply:
> Can you move this comment above the the delay /=10;

Sure. `npm run format` did it that way… Probably a bug in `prettier`?

Reply:
I personally prefer without `{}`, wherever possible anyway :)

Also, _[Biome](https://biomejs.dev/formatter)_ > [eslint-stylistic `--fix`](https://antfu.me/posts/why-not-prettier) > Prettier